types/react-select/src/Select.d.ts

```diff
@@ -197,7 +197,7 @@ export interface NamedProps<
   /* Status to relay to screen readers */
   screenReaderStatus?: (obj: { count: number }) => string;
   /* Style modifier methods */
-  styles?: StylesConfig;
+  styles?: StylesConfig<OptionType, IsMulti>;
```

Comment:
I suppose this technically would be a breaking change, but it feels incorrect to set default values; however, it may cause no harm in style methods?